x-pack/plugins/security_solution/public/timelines/components/timeline/notes_tab_content/index.tsx

```diff
@@ -122,6 +127,12 @@ interface NotesTabContentProps {
 
 const NotesTabContentComponent: React.FC<NotesTabContentProps> = ({ timelineId }) => {
   const dispatch = useDispatch();
+
+  const getScrollToTop = useMemo(() => getScrollToTopSelector(), []);
+  const scrollToTop = useShallowEqualSelector((state) => getScrollToTop(state, timelineId));
+
+  useScrollToTop('#scrollableNotes', !!scrollToTop);
```

Comment:
I wish I could utilize `useRef` here. But the container is an instance of `EuiFlexItem`, which doesn't accept the `ref` prop.